src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx

```diff
@@ -0,0 +1,204 @@
+
+Using telemetry in context functionality will increase your data ingest. Depending on your account's pricing model, this may have an impact on your ingest limits and billing.
+
+<Callout variant="caution">
+    If you want to use our telemetry in context solution and you already have an existing log forwarding solution, you must disable it. Otherwise you will be sending duplicate lines, which results in double billing. For more information, follow standard procedures to disable your [specific log forwarder](/docs/logs/forward-logs/enable-log-management-new-relic#log-forwarding).
+</Callout>
+
+**Example:** Your engineering team is troubleshooting a problem with your app, so you want to temporarily raise the limits to maximum data collection in your agent's config file. (By default, the maximum number of logs sampled by your APM agent is approximately 10,000 per minute.) But if you leave higher limits running for several days, this could result in a lot of unnecessary data and wasted money.
+
+To avoid any surprises, we recommend that you use [NRQL queries](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts/#data-queries) to create [alert conditions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts/#alerts) to keep track of your ingest limits. For example:
+
+<CollapserGroup>
+  
+  <Collapser
+    id="limits-ui"
+    title="View data limits and usage in UI"
+  >
+
+  To [review your data limits in the UI](/docs/data-apis/manage-data/view-system-limits): From the [account dropdown](/docs/glossary/glossary/#account-dropdown) in the New Relic One UI, click **Manage your data > Limits**. Or, to view your usage and estimated cost to date, click **View your usage**.
+
+  ![View data and usage from UI](./images/view-limits.png "View data and usage from UI")
+  
+  </Collapser>
+
+  <Collapser
+    id="query-estimate"
+    title="NRQL query: estimated cost example"
+  >
+
+  To query your [month-to-date estimated cost of your ingested data](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts/#month-cost), run this NRQL query:
+
+  ```
+  FROM NrMTDConsumption SELECT latest(estimatedCost) WHERE productLine = 'DataPlatform' SINCE this month
+  ```
+
+  </Collapser>
+
+  <Collapser
+    id="nrql-alert"
+    title="NRQL alert: usage threshold example"
+  >
+
+  To create an alert when your usage exceeds a fixed monthly threshold for gigabytes of data, add this [NRQL query](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/) to your alert condition:
+
+  ```
+  FROM NrMTDConsumption SELECT latest(GigabytesIngested) WHERE productLine = 'DataPlatform'
+  ```
+
+  </Collapser>
+
+</CollapserGroup>
+
+## Example: Troubleshooting poor response time and rising error rates [#response-time-example]
+
+Here is a detailed use case of using telemetry data in context to get to the root cause of a problem:
```

Comment:
Finish with "." instead of ":"? Totally up to you!

Reply:
Old habits die hard. :-) TIme to change, thanks!